internal/cli/main.go

```diff
@@ -180,30 +180,27 @@ func Commands(
 				baseCommand: baseCommand,
 			}, nil
 		},
-
```

Comment:
I noticed an inconsistency in the spacing between these commands, but also an apparent pattern of a space before a chunk of commands relating to the same operation that got broken at some point. So I made the code prettier to follow that pattern.